### PR TITLE
[cleanup] Mass replace tabs with spaces

### PR DESCRIPTION
**Pull request by sriyerg**
Mass search and replace of tabs with 2 spaces. This makes diffs easier. This is done on the entire repo. Image files, vendor directories and Makefiles are skipped. 

Signed-off-by: Srikrishna Iyer <sriyer@google.com>

### CODE REVIEW

**Review by weicaiyang — APPROVED** [review]
LGTM

**Comment by rswarbrick** [comment]
Thanks for doing this (and sorry for the assembly snippets with the tabs!). I wonder whether we should add a CI check for this sort of thing. What command did you run to get the list of files?

**Review by rswarbrick — APPROVED** [review]

**Comment by sriyerg** [comment]
> Thanks for doing this (and sorry for the assembly snippets with the tabs!). I wonder whether we should add a CI check for this sort of thing. What command did you run to get the list of files?

Nothing special! Just `find . -type f | xargs sed -i 's/\t/  /g'`. And manually restore images, vendor directories and makefiles. It was surprisingly not that painful. I am sure there is a better strategy than this though. A CI check would indeed be quite useful.

File: hw/ip/usbdev/pmod/dualpmod-kicad/dualpmod.sch [hunk 1]
```diff
@@ -41,8 +41,8 @@ F 0 "R4" V 3695 2300 50  0000 C CNN
 F 1 "22R" V 3786 2300 50  0000 C CNN
 F 2 "Resistor_SMD:R_0805_2012Metric_Pad1.15x1.40mm_HandSolder" H 3900 2300 50  0001 C CNN
 F 3 "~" H 3900 2300 50  0001 C CNN
-	1    3900 2300
-	0    1    1    0   
+  1    3900 2300
+  0    1    1    0
```
Comment (imphil) [inline hw/ip/usbdev/pmod/dualpmod-kicad/dualpmod.sch:45]: Are you sure these files still open correctly without tabs?

Comment (sriyerg) [reply hw/ip/usbdev/pmod/dualpmod-kicad/dualpmod.sch:45]: I am not. Any idea how to check? I will restore this file and re-push.

**Pull request merged**